### PR TITLE
Feat/login-signup

### PR DESCRIPTION
**Pull request by kphutiyagae**
* Implemented reactive forms for login and signup components.
* Added modal for user feedback and Validators.
* Added responsive design to components.

### CODE REVIEW

File: src/app/environments/environment.ts
Comment (CharelMarais) [inline src/app/environments/environment.ts:1]: What do we have here, some sensitive information...

**Review by CharelMarais — CHANGES_REQUESTED** [review]
The code looks good but you have your environment file showing.

File: src/app/environments/environment.ts
Comment (kphutiyagae) [reply src/app/environments/environment.ts:1]: Haha!
Git keeps tracking this file. Anyways, information was regenerated and removed from Git as requested.

**Review by CharelMarais — APPROVED** [review]
There we go

**Pull request merged**